### PR TITLE
Update recipes for Cypress v6.1.0 release

### PR DESCRIPTION
**Pull request by bahmutov**
Land after upgrading to Cypress v6.1.0

- [x] enable spying on image requests that return 304 from the server https://github.com/cypress-io/cypress/issues/8934
- [x] added spying and stubbing of redirect responses from the server

Screen Shot 2020-12-02 at 4 33 07 PM

### CODE REVIEW

File: examples/stubbing-spying__intercept/cypress/integration/redirect-spec.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    // stub the redirect call and once it returns from the server
+    // verify the target location and then change it to a safe "/"
+    cy.intercept('/getout', (req) => {
+      req.reply((res) => {
+        expect(res.statusCode).to.equal(302)
+        // the server wants to redirect us to another domain
+        expect(res.headers).to.have.property('location', 'https://www.cypress.io')
+        res.headers.location = '/'
+        // need to provide something for the updated "res"
+        // object to be used
```
Comment (flotwig) [inline examples/stubbing-spying__intercept/cypress/integration/redirect-spec.js:29]: hmm, this shouldn't be the case

**Review by flotwig — DISMISSED** [review]

**Comment by bahmutov** [comment]
Did not work otherwise ;)

**Comment by flotwig** [comment]
strange, setting the body should work like this: https://github.com/cypress-io/cypress/blob/2acdd7dedfde95ad80365dda2322f4a5b086986d/packages/driver/cypress/integration/commands/net_stubbing_spec.ts#L556-L570

if it doesn't work the same for headers, that's a bug

File: examples/stubbing-spying__intercept/cypress/integration/redirect-spec.js (hunk 1 above)
Comment (bahmutov) [reply examples/stubbing-spying__intercept/cypress/integration/redirect-spec.js:29]: definitely happens, I have opened https://github.com/cypress-io/cypress/issues/9555

**Pull request merged**